src/javascript/app/Modules/Trading/Containers/trade.jsx

```diff
@@ -11,6 +11,7 @@ import Test                 from './test.jsx';
 import FormLayout           from '../Components/Form/form-layout.jsx';
 import Digits               from '../../Contract/Containers/digits.jsx';
 import InfoBox              from '../../Contract/Containers/info-box.jsx';
+import { isDigitTradeType } from '../Helpers/digits';
```

Comment:
should be on line 12? but never mind let's merge it :D

Reply:
I'll adjust it in a separate PR.